README.md

```diff
@@ -6,6 +6,14 @@ A prometheus client for node.js that supports histogram, summaries, gauges and c
 
 See example folder for a sample usage. The library does not bundle any web framework, to expose the metrics just return the metrics() function in the registry.
 
+#### Usage with Node.js's `cluster` module
+
+Node.js's `cluster` module spawns multiple processes and hands off socket connections to those workers. Returning metrics from a worker's local registry will only reveal that worker's metrics, which is generally undesirable.
+
+To solve this, you can aggregate all of the workers' metrics in the master process. See example folder for a sample usage. By default, custom metrics are summed across workers, and default metrics use sensible defaults. To use a different aggregation method, define an `aggregate()` function for the metric's name in the global registry in the *master* process. (See the `AggregatorFactory` in `lib/cluster.js` for details.)
+
+Note: You must use the default global registry in cluster mode.
```

Comment:
In the current arrangement, the cluster master is the one that fulfills the metrics request by querying all of the workers and aggregating their local metrics. The workers need to know *which* registry to send their local metrics from, so the global registry is easy. I'll try to figure out a nice way to support other registries.